packages/preact-iso/router.js

```diff
@@ -18,6 +18,28 @@ const UPDATE = (state, url, push) => {
 	return url;
 };
 
+const exec = (url, route, matches) => {
+  url = url.trim('/').split('/');
```

Comment:
Is this the same function as in `/lib/loc.js`?

Reply:
It is yes, this is duplicated in demo due to testing I guess, this was the case before this PR